lib/chef-dk/command/verify.rb

```diff
@@ -79,8 +79,14 @@ def components
         # We can't run it right now since graphviz specs are included in the
         # test suite by default. We will be able to switch to that command when/if
         # Graphviz is added to omnibus.
-        c.unit_test { sh("bundle exec rspec --color --format progress spec/unit --tag ~graphviz") }
-        c.integration_test { sh("bundle exec cucumber --color --format progress --tags ~@no_run --tags ~@spawn --tags ~@graphviz --strict") }
+        c.unit_test do
+          sh("bundle install")
```

Comment:
So how are we going to wind up running tests?  The issue that we had in the past was that we'd wind up doing a bundle install into the omnibus builders /usr/local/bin/ruby and then running bundle exec rspec against that bundle in that ruby.   That created all kinds of issues since the ruby was different than what we shipped, and the gem resolution was not what we shipped.  One problem is that gems would go into the omnibus build and then we'd later re-bundle and slurp up a point release that fixed something horrible and then tests would pass and then we'd ship the brokenness...   So what is the story for how we avoid that?